libs/protocol-client/src/lib/graphql/queries.graphql

```diff
@@ -445,10 +445,10 @@ query getContributionCountByDateForUserInRange(
   }
 }
 
-query getContributionCountForDaoByActivityTypeInRange(
+query getContributionCountByActivityType(
   $where: GetDaoContributionCountInput!
```

Comment:
Should this be `GetContributionInput` instead of `GetDaoContributionCountInput` due to the name change? I was getting an error and noticed the different name